Formula/wireguard-go.rb

```diff
@@ -0,0 +1,25 @@
+class WireguardGo < Formula
+  desc "Userspace Go implementation of WireGuard"
+  homepage "https://www.wireguard.com/"
+  url "https://git.zx2c4.com/wireguard-go/snapshot/wireguard-go-0.0.20180514.tar.xz"
+  sha256 "a58599267359dcbb529945a41236f5fdf0971b896b994a1a3cd02180dd0da826"
+
+  depends_on "go" => :build
+  depends_on "dep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"gopath/src"
```

Comment:
`.install` below already takes care of the `mkdir -p` part

Reply:
Thanks. Left-over and missed it when committing. Fixed now.